ImisRestApi/Escape/Payment/Controllers/PaymentController.cs

```diff
@@ -146,9 +146,26 @@ public async Task<IActionResult> GetReconciliation()
         [Route("api/Reconciliation")]
         public IActionResult Reconciliation(int daysAgo)
         {
-            var done = imisPayment.RequestReconciliationReport(daysAgo);
-
-            return Ok(done);
+            object done = new { resp = "" };
+            // Make loop for all product from database that have account follow SP[0-9]{3} and do the function for all sp codes
+            var productsSPCodes = imisPayment.GetProductsSPCode();
+            if (productsSPCodes != null)
+            {
+                foreach (String productSPCode in productsSPCodes)
+                {
+                    done = imisPayment.RequestReconciliationReport(daysAgo, productSPCode);
+                }
+            }
+            System.Reflection.PropertyInfo pi = done.GetType().GetProperty("resp");
```

Comment:
you take only the latest answer, you could have error and success incrementor in the loop, the the message could have 2 section errors and done

Reply:
Ok. You are right. I will fix this.